### ISSUE TITLE
Security Vulnerability

### ISSUE THREAD

**Issue by phayes** [panicbit/recaptcha-rs#5]
There is currently a security vulnerability in `recaptcha-rs` via it's transitive dependency on `http 0.1.18`

Here are the details:

```
ID:       RUSTSEC-2019-0033
Crate:    http
Version:  0.1.18
Date:     2019-11-16
URL:      https://rustsec.org/advisories/RUSTSEC-2019-0033
Title:    Integer Overflow in HeaderMap::reserve() can cause Denial of Service

ID:       RUSTSEC-2019-0034
Crate:    http
Version:  0.1.18
Date:     2019-11-16
URL:      https://rustsec.org/advisories/RUSTSEC-2019-0034
Title:    HeaderMap::Drain API is unsound

Solution:  upgrade to >= 0.1.20
Dependency tree:
http 0.1.18
├── reqwest 0.9.24
    └── recaptcha 0.3.1
```

This can be fixed by updating to the latest version of reqwest (which will also necessitate creating sync and async functions within `recaptcha-rs`).

**Comment by panicbit** [comment]
I'll take care of updating the crate within the next days (latest on sunday), thanks for letting me know!

**Comment by panicbit** [comment]
I've taken care of this and published it as `0.4.0`.
`verify` is async now.
Fixed by 1fe51c056f25f045998439363d942cc158e084af.
